libsentrykube/quickpatch.py

```diff
@@ -92,6 +91,27 @@ def get_arguments(service: str, patch: str) -> Sequence[str]:
     return patch_data["schema"].get("required", [])
 
 
+def patch_json(patch_data: List[dict], resource_data: dict) -> dict:
+    """
+    Applies the patch to the json data
+    Assumes the patch & json data have a hierarchy of nested dictionaries
+    """
+    for patch in patch_data:
+        data = resource_data
+        op = patch.get("op")
+        path: str = patch.get("path", "")
+        value = patch.get("value")
+        if op == "replace" and path != "":
+            paths = path.strip("/").split("/")
+            for path in paths[:-1]:
+                if path not in data:
+                    data[path] = {}
+                data = data[path]
+            data[paths[-1]] = value
```

Comment:
I think you should document the semantics where you replace/merge a dictionary with another one.
Example:
```
resource = {
  'asd': {
      'sdf1': 1
   }
}

patch:
 - path: /asd
 - value: {sdf2: 3}
  
Should merge the two values with a deepmerge generating or should it replace the dictionary? If you replace it, please document it and explain how to add keys.
{
  sdf1: 1
  sdf2: 3
}
```